src/Accounts/Accounts/Utilities/Base64UrlHelpers.cs

```diff
@@ -0,0 +1,96 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Globalization;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.Profile.Utilities
+{
+    //The source code is copied from https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/master/src/client/Microsoft.Identity.Client/Utils/Base64UrlHelpers.cs
+    internal static class Base64UrlHelpers
```

Comment:
There's a very similar class in KeyVault: https://github.com/Azure/azure-powershell/blob/master/src/KeyVault/KeyVault/SecurityDomain/Models/Base64UrlEncoder.cs

I think we can move it to common lib

Reply:
Will log an issue to dedup in next release